code/modules/admin/secrets.dm

```diff
@@ -590,60 +590,6 @@
 				purrbation.")
 			log_admin("[key_name(usr)] has removed everyone from purrbation.")
 
-		if("flipmovement")
-			if(!check_rights(R_FUN))
-				return
-			if(alert("Flip all movement controls?","Confirm","Yes","Cancel") == "Cancel")
```

Comment:
Undocumented change.

Why are you removing these admin buttons?

[`You are going to be expected to document all your changes in the pull request. Failing to do so will mean delaying it as we will have to question why you made the change`](https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#pull-request-process)

Please mention this change in the PR body, (as well as explain how its related to this pull request). Thank you

Reply:
Sorry for that, updated OP to include the removal. I removed it because I've never seen anyone use this on Yogstation and I couldn't be arsed to spend much time properly fixing such a niche button.

Reply:
Don't some chemicals or situations other that admin buttons cause movement flipping?

There's something that does it, I just don't recall what.